topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorClient.java

```diff
@@ -59,6 +62,7 @@ public class TopicGeneratorClient {
     private static final int MAX_CONTEXT_LENGTH = 16385;
     private static final EncodingRegistry REGISTRY = Encodings.newDefaultEncodingRegistry();
     private static final Encoding ENCODING = REGISTRY.getEncodingForModel(AI_MODEL);
+    private final List<StringFilter> stringFilters = Lists.newArrayList(new ChuckNorrisFilter("en"), new ChuckNorrisFilter("fr-CA-u-sd-caqc"));
```

Comment:
LOL, I was wondering how you were going to implement this without committing a bunch of foul language to our repo. :)